### ISSUE TITLE
[WFRP4e System] Using Called Shots gives an unremovable -20 modifier. 

### ISSUE THREAD

**Issue by moo-man** [moo-man/WFRP4e-FoundryVTT#1248]
When applying Called Shots (without having relevant talents that may negate the penalty), the roll dialog applies a -20 penalty. It is, however, impossible to remove that penalty.

Example: Player 1 attacks an enemy. He selects the option "Head", which applies a -20 penalty to the modifier. For any reason, he is given permission to ignore this -20 penalty. Inputting "0" in the modifier box gives "-20". 

Basically, every single other prefilled modifier in the roll dialog can be freely modified, but Called Shots modifiers always apply.<br/>**From**: The First#7330<br/>foundry: 10.285<br/>wfrp4e: 6.1.2

**Comment by Txus5012** [comment]
Type "20" and it will be modified to "0".
